configs/dsdl/voc.py

```diff
@@ -0,0 +1,65 @@
+_base_ = [
+    '../_base_/models/deeplabv3_r50-d8.py', '../_base_/default_runtime.py',
+    '../_base_/schedules/schedule_20k.py'
+]
+
+# dataset settings
+dataset_type = 'DSDLSegDataset'
+data_root = 'data/VOC12-seg'
+img_prefix = 'original'
```

Comment:
Is 'original' a keyword needed inside DSDL?

Reply:
no, users should change it by actual path. (or no img_prefix, decied by it's file sturct)